addons/packages/pinniped/0.12.1/bundle/.imgpkg/images.yml

```diff
@@ -9,11 +9,11 @@ images:
           url: docker.io/getpinniped/pinniped-server:v0.12.1
   image: index.docker.io/getpinniped/pinniped-server@sha256:8b4ee3b279d8d1d1f1c65d95f8611a99e00c6d2fbb5dbf974ad76ac4ca563d73
 - annotations:
-    kbld.carvel.dev/id: projects.registry.vmware.com/tce/dex:v2.30.2_vmware.1
+    kbld.carvel.dev/id: projects.registry.vmware.com/tce/dex:v2.31.0_vmware.1
```

Comment:
did you regenerate this file with the command invocation here: https://github.com/vmware-tanzu/community-edition/blob/62fa9d7dc5107dfe2754e483881c0c0e8dd18d7d/addons/packages/pinniped/0.12.0/README.md#generate-image-package

i think i would have expected the `image:` to change. this is that chicken-and-egg thing that absolutely drives me up the wall. what if we just used the oss dex image here so we didn't have to worry about pushing it?

Reply:
Updated, ran `kbld` several times, the `kbld.carvel.dev/origins` `resolved` bits are being wiped out, not sure how important that is. 🤔

Reply:
ok better, diff due to `kbld` 0.30 -> 0.32